src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -139,9 +132,9 @@ type TrackErrors = (errors: string[]) => void
   Background:
     https://github.com/jaredpalmer/formik/issues/1484#issuecomment-490558973
  */
-const OnSubmitValidationError: React.FC<{
+export const OnSubmitValidationError: React.FC<{
   cb: TrackErrors
-  formikProps: FormikProps<FormValues>
+  formikProps: Partial<FormikProps<FormValues>>
```

Comment:
I wonder if this form should be moved up to a more common/shared directory- it seems like other forms may want to use it as well.

Reply:
Agreed - there are so many things shared across `Register` and `ConfirmBid` and they have imports from each other, which makes it harder to trace dependencies. Though that would probably be a number of incremental changes, so I wanted to focus on shipping first. cc @dblandin